app/code/Magento/NewsletterApi/etc/webapi.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
```

Comment:
Here you could also add:
* Endpoints for the customer to subscribe/unsubscribe himself, so no admin privileges needed.
* Guest resources
* More of the API, e. g. subscribe by id, unsubscribe, etc.